lively.ide/components/change-tracker.js

```diff
@@ -105,6 +106,7 @@ export class ComponentChangeTracker {
     if (change.selector === 'addMorphAt' && change.target.textAndAttributes?.includes(change.args[0])) return true;
     if (!change.selector &&
         change.prop !== 'layout' &&
+        change.prop !== 'vertices' &&
```

Comment:
What are these two for? :thinking:

Reply:
This case is for properties where the values are often updated in place. In those situtations `change.prevValue` and `change.value` reference exactly the same object and neither `obj.equals` nor `obj.deepEquals` helps us much. Right now I found this affects `layout` and `vertices`, however we may wanna manage this in a more canonical way in the future. We once had the policy in mind to *never* update properties in place, however that turned out to be not ideal for performance reasons, so we moved away from that idea.